src/dialogs/rename.tsx

```diff
@@ -57,47 +76,58 @@ const RenameItemModal = ({ dialogResult, path, selected } : {
     const [name, setName] = useState(selected.name);
     const [nameError, setNameError] = useState<string | null>(null);
     const [errorMessage, setErrorMessage] = useState<string | undefined>(undefined);
+    const [overrideFileName, setOverrideFileName] = useState(false);
 
-    let title;
-    if (selected.type === "reg") {
-        title = cockpit.format(_("Rename file $0"), selected.name);
-    } else if (selected.type === "lnk") {
-        title = cockpit.format(_("Rename link $0"), selected.name);
-    } else if (selected.type === "dir") {
-        title = cockpit.format(_("Rename directory $0"), selected.name);
-    } else {
-        title = _("Rename $0", selected.name);
-    }
-
-    const renameItem = () => {
+    const renameItem = (force = false) => {
         const newPath = path.join("/") + "/" + name;
+        const mvCmd = ["mv", "--no-target-directory"];
+        if (force) {
+            mvCmd.push("--force");
+        }
+        mvCmd.push(path.join("/") + "/" + selected.name, newPath);
 
-        cockpit.spawn(["mv", "--no-target-directory", path.join("/") + "/" + selected.name, newPath],
-                      { superuser: "try", err: "message" })
+        cockpit.spawn(mvCmd, { superuser: "try", err: "message" })
                 .then(() => {
                     dialogResult.resolve();
                 }, err => setErrorMessage(err.message));
     };
 
+    const title = (
+        <>
+            {_("Rename") + " "}
```

Comment:
The design shows a question mark `Rename <b>Foo</b>?` here, I would suggest using `fmt_to_fragments`.